package.json

```diff
@@ -53,7 +53,7 @@
   },
   "scripts": {
     "generate": "node script/generate-react",
-    "format": "remark . -qfo && prettier --write '**/*.js' && xo --fix",
+    "format": "remark . -qfo && prettier --write \"**/*.js\" && xo --fix",
```

Comment:
Do single quotes not work? 🤔

Reply:
No, it's a common behaviour. 

* https://github.com/keithamus/npm-scripts-example/issues/5
* https://github.com/Jephuff/fix-windows-single-quotes#readme